components/ui/bar/NavBar.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Text, View } from 'react-native';
+
+interface NavBarProps {
+  title: string;
+  type: 'normal' | 'segment';
+}
+
+const NavBar: React.FC<NavBarProps> = ({ title, type }) => {
+  const textSize = type === 'normal' ? 'text-title2' : 'text-title3';
+  const paddingX = type === 'normal' ? 'px-6' : '';
```

Comment:
if we use paddingX and paddingY in one place could we merga that two ifs to one with one output?
type === 'normal' ? 'px-6' : '';